RecoParticleFlow/PFClusterProducer/src/SpikeAndDoubleSpikeCleaner.cc

```diff
@@ -117,10 +117,55 @@ clean(const edm::Handle<reco::PFRecHitCollection>& input,
     }    
     const spike_cleaning& clean = _thresholds.find(hitlayer)->second;    
     if( rechit.energy() < clean._singleSpikeThresh ) continue;
+
+
+
+    //Fix needed for HF
+    float compsumE = 0.0;
+    if ((hitlayer==12 or hitlayer==11))
+	{
+	int predphi =2;
+
+	int comp = std::abs(hitlayer-13);
```

Comment:
"13" here is quite likely "12+1", where "12" can be extracted from some enum or configured list